modules/react/combobox/lib/ComboboxPopper.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {usePopupPopper, Popper} from '@workday/canvas-kit-react/popup';
+
+import {Menu, useMenuModel} from '@workday/canvas-kit-react/menu';
+
+export interface MenuPopperProps extends ExtractProps<typeof Popper> {}
+
+export const useMenuPopper = usePopupPopper;
+
+export const ComboboxMenuPopper = createSubcomponent('div')({
+  displayName: 'Combobox.Menu.Popper',
+  modelHook: useMenuModel,
+  elemPropsHook: useMenuPopper,
+})<MenuPopperProps>(({children, ...elemProps}) => {
+  return (
+    <Menu.Popper fallbackPlacements={['top', 'bottom']} {...elemProps}>
```

Comment:
I specifically created this file to update the fallback placements. The popper should only open at the top or bottom, otherwise the fallbacks include left and right and this is not intended behavior